openedx/features/caliper_tracking/tests/expected/edx.course.enrollment.activated.json

```diff
@@ -0,0 +1,41 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "id": "urn:uuid:2635b9dd-0061-4059-ac61-2718ab366f75",
+  "type": "Event",
+  "actor": {
+    "id": "edx.devstack.lms:18000/u/ali",
+    "type": "Person"
+  },
+  "action": "Activated",
+  "object": {
+    "id": "course-v1:edx+cs-101+2018",
+    "type": "CourseOffering",
+    "username": "ali"
+  },
+  "eventTime": "2018-11-01T03:35:04.214775",
+  "referer": "http://localhost:18000/courses/course-v1:edx+cs-101+2018/about",
+  "session": "8205ce7ea40a0619955e15f072dc33e2",
```

Comment:
This value of `session` is invalid according to caliper
https://www.imsglobal.org/sites/default/files/caliper/v1p1/caliper-spec-v1p1/caliper-spec-v1p1.html#event
Please move this field to `extra_fields`

Reply:
moved.